pkg/lib/cockpit.js

```diff
@@ -2618,6 +2618,10 @@ function factory() {
             return _("Cockpit could not contact the given host.");
         else if (problem == "too-large")
             return _("Too much data");
+        else if (problem == "not-found")
+            return _("No such file or directory");
```

Comment:
This is *not* how `not-found` is [defined](https://cockpit-project.org/guide/latest/cockpit-error.html). We use it for D-Bus, PackageKit transactions, etc. This error *really* needs to be handled on the caller side which has all the necessary context to build an useful error mesage.